lib/ImagePrototypeInit.js

```diff
@@ -47,6 +47,19 @@
         this.__locked = false;
     };
 
+    Image.prototype.getMetadata = function() {
+        if (!this.__metadata || this.__metadata === "") {
+            return null;
+        } else {
+            return this.__metadata;
+        }
+    };
+
+    Image.prototype.setMetadata = function(data) {
+        this.__metadata = data;
```

Comment:
How are different types of 'data' handled?

Eventually this value will go all the way to [here](https://github.com/joannajw/lwip/blob/png-metadata/src/encoder/init.cpp#L35), at which time it will casted to a string.

If users pass here an object, for example, do they expect us to serialize it for them? What if they pass in a function?

In the docs we say that 'data' should be a string. So we should probably enforce it here by doing type checks and throwing an exception if needed, as done in other methods.